### ISSUE TITLE
Accessibility: Tab order Focus is not in Correct in Date Picker Pallet.

### ISSUE THREAD

**Issue by RamojiraoMuppirisetti** [ng-bootstrap/ng-bootstrap#4534]
### Bug description:
When open the Date Picker pallet and see the tab order focus, Tab order focus should be inside the pallet and when press the esc key then only focus should come out from pallet.

Angular: 8

ng-bootstrap: 5.1.4

**Comment by maxokorokov** [comment]
@RamojiraoMuppirisetti current ng-bootstrap version is `15.1.0`, please check with the latest version. I don't think we'll be able to fix anything in such an old version.

**Comment by maxokorokov** [comment]
Closing as no response, please reopen if necessary.
